### DISCUSSION TITLE
How to show brightness control band7

### DISCUSSION THREAD

**Discussion by miguelavh** [zepp-health/zeppos-forum#227] [Q&A] [answered]
Hello, I am making a watchface for band7, and I would like to add a direct access to the brightness control to be able to control it directly from the watchface, I have seen that it is possible to call applications with appStart but I don't know the url of the brightness control app, any ideas? Do you know that url to make the call directly? It could be something like hmApp.startApp({ url: 'pai_app_Screen', native: true }), changing pai_app_Screen for brightness_app_Screen or similar... Thank you in advance

**Comment by SashaCX75** [comment]
I think you should read here
https://docs.zepp.com/docs/1.0/watchface/api/hmSetting/getScreenAutoBright/
and here
https://docs.zepp.com/docs/1.0/watchface/api/hmSetting/setBrightness/

**Reply by miguelavh** [reply]
It works, but It's not that I'm searching for, I'd like to show brigthness control bar, to control brightness with a slide...

**Reply by silver-zepp** [reply]
You can just store the value you set with `setBrightness(...)` into your own variable and show that value on the screen where you need it.

And for the link to the brightness settings, I think it's `hmApp.startApp({url: "Settings_displayBrightScreen", native: true}) `

**Reply by miguelavh** [reply]
It almost works correctly, it takes me to the previous screen where I want to go, I want to go directly to the slide screen "Brightness adjustment" and it leaves me in the "Display & Brightness" screen, can you indicate link to "Brightness adjustment" screen?
Thank you in advance

**Reply by miguelavh** [reply, marked as answer]
I found the url... 

hmApp.startApp({url: "Settings_lightAdjustScreen", native: true})

Thank you

**Reply by silver-zepp** [reply]
Nicely done @miguelavh !
